app/src/main/java/com/rafaelmardom/movies_and_shows_api/domain/GetMovieDetailUseCase.kt

```diff
@@ -0,0 +1,30 @@
+package com.rafaelmardom.movies_and_shows_api.domain
+
+class GetMovieDetailUseCase (private val movieRepository: MovieRepository) {
+
+    fun execute (movieId: String) : MovieDetail {
+        val movie = movieRepository.getMovieById(movieId)
+
+        return MovieDetail(
+            movie.id,
+            movie.title,
+            movie.plot,
+            movie.genre,
+            movie.rating,
+            movie.year,
+            movie.poster
+            )
+
+    }
+
+}
+
+data class MovieDetail (
```

Comment:
MovieDetail is  business model??

Reply:
I think I don't understand the question, but I've moved the data class inside of UseCase the same way I've done with MovieFeed below.